org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizardPage.java

```diff
@@ -258,6 +280,8 @@ public boolean isPageComplete() {
 			locationError = "Unable to create such directory";
 		} else if (directory.exists() && !directory.canWrite()) {
 			locationError = "Cannot write in this directory";
+		} else if (getTemplate().equals("")) {
```

Comment:
This mustn't be an error because dotnet-core 1.1 allows empty template (as it doesn't have templates). You should replace it with something like `if (!templates.isEmpty() && getTemplate().isEmpty()) { templateError =...`. That should work with both versions of dotnet.